src/viewer.test.tsx

```diff
@@ -180,52 +205,16 @@ describe("SeqViz rendering (React)", () => {
 
     // @ts-expect-error ts-migrate(2322) FIXME: Type '{ seq: string; viewer: "linear"; name: strin... Remove this comment to see the full error message
     const wrapper = mount(<SeqViz {...defaultProps} seq={seq} viewer="linear" />);
-    await wrapper.instance().componentDidMount();
+    const _componentDidMount = wrapper.instance().componentDidMount;
```

Comment:
Nbd but wouldn't these tests fail either way? (With undefined function if null)

Reply:
This resolves a typescript error - as you say it doesn't change the functionality of the test.